packages/components/addon/components/hds/table/index.hbs

```diff
@@ -54,7 +66,14 @@
       {{#each (sort-by this.getSortCriteria @model) key=this.identityKey as |record|}}
         {{yield
           (hash
-            Tr=(component "hds/table/tr")
+            Tr=(component
+              "hds/table/tr"
+              isSelectable=@isSelectable
+              didInsert=this.addCheckbox
```

Comment:
I can't find the `addCheckbox` function, maybe these `didInsert` are not needed anymore?